spec/system/sign_up_spec.rb

```diff
@@ -5,9 +5,15 @@
     visit '/users/sign-up'
   end
 
+  
+
   context 'when user does not exist' do
     let(:user) { create(:user) }
 
+    it 'has service name' do
```

Comment:
I'd opt for a unit test on the helper itself. Thing about these `have_text` assertions is we don't know where exactly the text is, so for example it could be catching something a content designer adds in YAML.